tools/ports/bullet.py

```diff
@@ -0,0 +1,48 @@
+import os, shutil, logging, subprocess, sys, stat
+
+TAG = 'version_1'
+
+def build_with_configure(ports, shared, path):
+  if not sys.platform.startswith('win'): #TODO: test on windows
```

Comment:
This function is not being used, I just left it as a reference (it can be used to generate config scripts for example).